spec/lib/task_helpers/exports/widgets_spec.rb

```diff
@@ -3,11 +3,13 @@
     Dir.mktmpdir('miq_exp_dir')
   end
 
+  let(:core_widget_dir) { File.expand_path(File.join(Rails.root, "product/dashboard/widgets")) }
+
   before do
     EvmSpecHelper.local_miq_server
 
     MiqReport.seed_report("Vendor and Guest OS")
-    MiqWidget.seed_widget("chart_vendor_and_guest_os")
+    MiqWidget.sync_from_file(Dir.glob(File.join(core_widget_dir, "*chart_vendor_and_guest_os*")).first)
```

Comment:
Since this is a single widget and we know the name I think you can just use the full path and drop the glob

```
let(:widget_path) { Rails.root.join("product/dashboard/widgets/chart_vendor_and_guest_os.yaml") }

MiqWidget.sync_from_file(widget_path)
```